core/src/main/java/org/jobrunr/scheduling/ScheduleExpressionType.java

```diff
@@ -30,5 +35,13 @@ public static Schedule getSchedule(String scheduleExpression) {
         throw new ScheduleException(scheduleExpression);
     }
 
+    public static String findSchedule(String cron, String interval) {
```

Comment:
I find that `findSchedule` doesn't describe well what the method does. What do you think of `selectConfiguredScheduleExpression`? Could also become more generic and accept any amount of parameters.